index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- hero section -->
+    <div id="home" class="home">
+      <div class="home-top">
+        <div class="top-content">
+          <img src="./assets/facebook.png" alt="facebook" />
+          <img src="./assets/twitter.png" alt="twitter" />
+          <span>Logout</span>
+          <span>MyPage</span>
+          <span>Korean</span>
+        </div>
+      </div>
+      <div class="toolBar">
+        <a href="./index.html"
+          ><img class="logo" src="./assets/medical-logo.png" alt="logo"
+        /></a>
+        <div class="toogle">
+          <img
+            class="mobile-menu"
+            src="./assets/mobile_menu.png"
+            alt="mobile menu"
+          />
+        </div>
+
+        <nav>
+          <ul class="navigation">
+            <li class="nav-item"><a href="./html/about.html">About</a></li>
+            <li class="nav-item"><a>Program</a></li>
+            <li class="nav-item"><a>Join</a></li>
+            <li class="nav-item"><a>Sponsor</a></li>
+            <li class="nav-item"><a>News</a></li>
+          </ul>
+          <img
+            class="close-menu"
+            src="./assets/close_mobile_menu.png"
+            alt="close"
+          />
+        </nav>
+        <a class="compain-btn">Make an Appointment</a>
+      </div>
+      <div class="home-content">
+        <div>
+          <span>"Hello! we are a private medical clinic"</span>
+          <h1>PROFESSIONAL MEDICAL CARE FOR YOUR ENTIRE FAMILY</h1>
+        </div>
```

Comment:
- [ ] As the previous reviewer mentioned this should follow the design guidelines, please know that on the original design, the font "COCOGOOSE" was used for the main title, kindly adjust that.

Reply:
Thank you  @mirouhml  for bringing this to my attention.  I will make sure to adjust the font to "COCOGOOSE"